app/src/main/kotlin/codes/chrishorner/socketweather/home/HomeForecastPresenter.kt

```diff
@@ -27,6 +33,16 @@ class HomeForecastPresenter(view: View) {
     currentIcon.setImageDrawable(context.getWeatherIconFor(forecast.iconDescriptor, forecast.night))
     currentTemp.text = forecast.currentTemp.formatAsDegrees(context)
     feelsLikeTemp.text = forecast.tempFeelsLike?.formatAsDegrees(context) ?: "--"
+    humidity.text = forecast.humidity?.formatAsPercent(context) ?: "--"
```

Comment:
If the humidity isn't available, how would you feel about making the view `GONE` instead?

Reply:
Good idea! Done.